src/main/java/me/partlysanestudios/partlysaneskies/HelpCommands.java

```diff
@@ -122,6 +122,8 @@ public static void printHelpMessage() {
                 "\n&b&l&nWelcome to Partly Sane Skies!&r" +
                 "\nPartly Sane Skies is a mod developed by Su386 and FlagMaster. This mod aims to be a quality of life mod for Hypixel Skyblock." +
                 "\n" +
+                "\n Running Version: &e" + PartlySaneSkies.VERSION +
```

Comment:
What if we had a /version command that printed all the version information

Reply:
Idk, I really like the way skyhanni handles that with showing the version in the config title 

You'd have to decide

Reply:
I think it's probably good